tools/generate-helpers/bootstrap-migration/seq_num.go.tpl

```diff
@@ -0,0 +1,11 @@
+package internal
```

Comment:
add generate comment to indicate this file should not be changed

Reply:
I'm not sure we can have this totally generated unless we provide a mechanism for passing in `MinimumSupportedDBVersionSeqNum` as that will diverge from `{{.nextSeqNum}}` and should in fact rarely be updated.